---
geometry_msgs/Point position
bool isTracked